### PR TITLE
Silence a bunch of expected errors and warnings in test logs

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: akka-http-tests/src/test/resources/application.conf [hunk 1]
```diff
@@ -6,4 +6,8 @@ akka {
     default-dispatcher.throughput = 1
   }
   stream.materializer.debug.fuzzing-mode = on
+
+  # silence some annoying warnings
+  stream.secret-test-fuzzing-warning-disable = 42
+  akka.actor.warn-about-java-serializer-usage = false
```
Comment (raboof) [inline akka-http-tests/src/test/resources/application.conf]: Spurious `akka.`?

File: akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala [hunk 2]
```diff
@@ -162,9 +177,11 @@ class FileAndResourceDirectivesSpec extends RoutingSpec with Inspectors with Ins
       shouldReject("\\")
       shouldReject("%5c")
       shouldReject("..%5c")
-      shouldReject("..%255c")
-      shouldReject("..%c0%af")
-      shouldReject("..%c1%9c")
+
+      // FIXME these don't cause log warnings for some reason
```
Comment (raboof) [inline akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala]: Perhaps the loglevel?

File: akka-http-tests/src/test/resources/application.conf (hunk 1 above)
Comment (johanandren) [reply akka-http-tests/src/test/resources/application.conf]: d'oh!

File: akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala (hunk 2 above)
Comment (johanandren) [reply akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala]: You mean some suspicious strings cause warnings but others cause info logging?

**Pull request merged**